checks/shell_download_validate_test.go

```diff
@@ -0,0 +1,84 @@
+// Copyright 2021 Security Scorecard Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package checks
+
+import (
+	"fmt"
+	"io/ioutil"
+	"testing"
+)
+
+func TestIsSupportedShellScriptFile(t *testing.T) {
+	t.Parallel()
+
+	tests := []struct {
+		filename string
```

Comment:
Nit: add a `name string` field here which can be used to identify the test. Currently we use `filename` to identify the test (line 66).